dispatch/static/manager/src/js/pages/Articles/ArticleIndexPage.js

```diff
@@ -46,28 +49,45 @@ function ArticlePageComponent(props) {
   const section = props.entities.sections[props.location.query.section]
   const title = section ? `${section.name} - Articles` : 'Articles'
 
+  const filters = (
+    <div className='c-item-list__header__filters'>
```

Comment:
To simplify this a little bit, filters could be an array of components: 

```jsx
const filters = [
  <SectionFilterInput
    selected={props.location.query.section}
    update={(section) => props.searchArticles(props.location.query.author, section, props.location.query.q)}
    />,
  <AuthorFilterInput
    selected={props.location.query.author}
    update={(author) => props.searchArticles(author, props.location.query.section, props.location.query.q)}
    />
]
```

And then the `ItemListHeader` component could wrap them in the div:

```jsx
<div className='c-item-list__header__filters'>{filters}</div>
```